elasticsearch/templates/ingress.yaml

```diff
@@ -1,38 +1,36 @@
 {{- if .Values.ingress.enabled -}}
-{{- $fullName := include "uname" . -}}
-{{- $servicePort := .Values.httpPort -}}
-{{- $ingressPath := .Values.ingress.path -}}
+{{- $fullName := include "elasticsearch.fullname" . -}}
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: {{ $fullName }}
   labels:
-    app: {{ .Chart.Name }}
```

Comment:
Removing these labels breaks backwards compatibility.

Reply:
Short term pain, long term gain? ;) Your call!

Reply:
Could you explain what the gain is? Just sounds like a lot of pain to me. We can add the new labels without removing the old ones which is a win win.